requirements.txt

```diff
@@ -48,6 +48,7 @@ typing-extensions==4.5.0
 
 # extras
 astor>=0.8.1
+algoliasearch>=3.0.0
```

Comment:
can we restrict the package version here as well

Reply:
Locked